llama_stack/providers/remote/inference/ollama/config.py

```diff
@@ -12,7 +13,7 @@
 
 
 class OllamaImplConfig(BaseModel):
-    url: str = DEFAULT_OLLAMA_URL
+    url: str = os.getenv("OLLAMA_URL", DEFAULT_OLLAMA_URL)
```

Comment:
why is this change necessary? I think we need to fix this https://github.com/meta-llama/llama-stack/blob/main/llama_stack/providers/tests/inference/fixtures.py#L96 instead because it is wrong (it still uses host / port instead of URL)